Week-9/demo-react-app/src/Home.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import './Home.css'
+
+function Note(props){
+
+    function handleCheck(id){
+        var col = document.getElementById(id), check = document.getElementById(id+"_done")
```

Comment:
This too create a seperate component like Note.js

Reply:
U should  not use document.getelemnt.. in react very bad practice.. make use of useref hook to access any dom elemnet